### ISSUE TITLE
Accessibility Audit: Bulk results uploader user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5776]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the bulk results uploader user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/results/upload/*`

Use cases to test:
* Upload spreadsheet
    * upload results in validation error(s)
    * upload success
    * toast if file is wrong format or empty
* View upload history (with no, single, and multiple uploads)
* View submission details (from upload history page)
* Bulk results upload guide (link from upload spreadsheets page)
* Device code lookup page (inc search with no results)

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by BobanL** [comment]
Accessibility tickets created: 
* #5859 
* #5858 

Other tickets created: 
* #5857 
* #5855
